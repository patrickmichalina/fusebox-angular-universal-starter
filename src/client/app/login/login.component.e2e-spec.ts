import { baseUrl, browser } from '../../../../tools/test/jest.e2e-setup'

describe('Login Page', () => {
  it('should have title', async () => {
    expect.assertions(1)

    const page = browser.goto(`${baseUrl}/login`)

    const text = await page.evaluate(() => document.title)

    expect(text).toContain('Login - Fusebox Angular Universal Starter')
  })
})
