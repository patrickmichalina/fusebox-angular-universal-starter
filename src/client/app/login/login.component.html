<div fxLayout="column" fxLayoutAlign="center center">
  <h1>Login</h1>
  <pm-login-card></pm-login-card>
</div>