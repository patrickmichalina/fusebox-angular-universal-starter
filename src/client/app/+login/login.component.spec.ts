import { LoginComponent } from './login.component'
import { async, ComponentFixture, TestBed } from '@angular/core/testing'
import { Component } from '@angular/core'
import { LoginModule } from './login.module'

describe(LoginComponent.name, () => {
  let fixture: ComponentFixture<LoginComponent>

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      imports: [LoginModule],
      declarations: [TestComponent]
    }).compileComponents()
  }))

  beforeEach(async(() => {
    fixture = TestBed.createComponent(LoginComponent)
  }))

  afterEach(async(() => {
    TestBed.resetTestingModule()
  }))

  it('should match snapshot', async(() => {
    expect(fixture).toMatchSnapshot()
  }))

  it('should compile', async(() => {
    expect(fixture.nativeElement).toBeDefined()
  }))
})

@Component({
  selector: 'test-component',
  template: '<pm-login></pm-login>'
})
class TestComponent {}
