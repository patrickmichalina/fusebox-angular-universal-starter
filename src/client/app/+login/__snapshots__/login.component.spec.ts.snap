// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginComponent should compile 1`] = `
<pm-login
  containerClass={[Function Boolean]}
>
  <h1>
    Login
  </h1>
  <pm-login-card>
    
    
  
    
  </pm-login-card>
</pm-login>
`;
